package.json

```diff
@@ -16,11 +16,13 @@
   "scripts": {
     "start": "NODE_ENV=production node ./src/server/server.babel.js",
     "dev": "NODE_ENV=development node ./src/server/server.babel.js",
-    "build": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\"",
+    "build": "rimraf build && npm run build:server && npm run build:client",
```

Comment:
It was in my check-in comment, but I'll re-add here: the client bundle depends on the existence of prerender.js which is built by the server.